autoload/rust.vim

```diff
@@ -359,6 +359,63 @@ function! s:system(pwd, cmd)
 	return system(cmd)
 endfunction
 
+" Playpen Support {{{1
+" Parts of gist.vim by Yasuhiro Matsumoto <mattn.jp@gmail.com> reused
+" gist.vim available under the BSD license, available at
+" http://github.com/mattn/gist-vim
+function! s:has_webapi()
+  if !exists("*webapi#http#post")
+    try
+      call webapi#http#post()
+    catch
+    endtry
+  endif
+  return exists("*webapi#http#post")
+endfunction
+
+function! rust#Play(count, line1, line2, ...) abort
+  redraw
+
+  if !exists('g:rust_playpen_url')
+    let l:rust_playpen_url = 'https://play.rust-lang.org/'
+  else
+    let l:rust_playpen_url = g:rust_playpen_url
+  endif
+  if !exists('g:rust_shortener_url')
+    let l:rust_shortener_url = 'https://is.gd/'
+  else
+    let l:rust_shortener_url = g:rust_shortener_url
+  endif
```

Comment:
FWIW, this can be expressed more simply as

```vim
let l:rust_playpen_url = get(g:, 'rust_playpen_url', 'https://play.rust-lang.org/')
let l:rust_shortener_url = get(g:, 'rust_shortener_url', 'https://is.gd/')
```